extensions/csrc/cuda/multi_tensor_l2norm_kernel.cu

```diff
@@ -12,14 +12,98 @@
 
 #include "multi_tensor_apply.cuh"
 #include "../common/micros.h"
-#include "include/block_reduce.h"
+#include "funcs/reduce_function.h"
 
 #define BLOCK_SIZE 512
 #define ILP 4
 
-using colossalAI::cuda::utils::block_reduce;
-using colossalAI::cuda::utils::reduce_block_into_lanes;
-using colossalAI::cuda::utils::reduce_block_into_lanes_max_op;
+
```

Comment:
Better to run relevant tests at training side.

Reply:
Done